consensus/parlia/parlia.go

```diff
@@ -91,6 +90,7 @@ var (
 		common.HexToAddress(systemcontracts.TokenHubContract):           true,
 		common.HexToAddress(systemcontracts.RelayerIncentivizeContract): true,
 		common.HexToAddress(systemcontracts.CrossChainContract):         true,
+		common.HexToAddress(systemcontracts.StakeHubContract):           true,
```

Comment:
It is not enough to only have `StakeHubContract`, because consensus will init the other three contract as well, so other three contracts should place here as well.

Reply:
Done